src/screens/Base.js

```diff
@@ -24,6 +26,7 @@ export const BaseScreen = ({ renderOn }) => {
 
     Header({ renderOn: '#app-header' });
     Footer({ renderOn: '#app-footer' });
+    Spinner({ renderOn: '#spinner-place', loading: true });
```

Comment:
loading: true? bruh, it's constantly spinning, it should only display while data from api is loading ;-;

Reply:
Resolved